cmd/gremlins.go

```diff
@@ -43,9 +58,40 @@ and friends.
 		Version: version,
 	}
 
-	cmd.AddCommand(newUnleashCmd().cmd)
+	configPaths := []string{
+		".",
+		"/etc/gremlins",
+		"$HOME/.gremlins",
```

Comment:
Should we add `$HOME/.config/gremlins/gremlins` which is XDG_CONFIG_HOME standard location?

Reply:
ok, will do